editor/editor_file_system.cpp

```diff
@@ -794,6 +794,17 @@ void EditorFileSystem::_scan_new_dir(EditorFileSystemDirectory *p_dir, DirAccess
 			}
 		}
 
+		// TODO-DOC: is it okey to call this here? some how the script
+		//  needs to be compiled to generate and cache the documentations.
```

Comment:
I'm notsure about this. Will have to test in a big project to see if there are issues or slowdowns. I think this runs in a thread, so it might not be so bad.